src/components/Referrals/referralsHelper.js

```diff
@@ -69,6 +76,26 @@ export const tierDiscountInfo = {
   2: 10,
 };
 
+export const totalRebateInfo = {
+  0: 10,
+  1: 20,
+  2: 25,
+};
```

Comment:
hm, we don't need to hardcode this, beacause we have useTiers hook which requests ReferralStorage for actual totalTebate config

Reply:
updated!